config/sync/field.field.paragraph.stanford_media_caption.su_media_caption_caption.yml

```diff
@@ -10,15 +10,13 @@ dependencies:
     - text
 third_party_settings:
   allowed_formats:
-    stanford_minimal_html: stanford_minimal_html
```

Comment:
This was a conscious decision. What's the use case to open the caption to all html?

Reply:
That's a good point. This changed when I did the config_export.  I'll revert it.

Reply:
Just waiting on this revert

Reply:
reverted